src/user-guide/compound-commands.rst

```diff
@@ -0,0 +1,28 @@
+.. _CompoundCommands:
+
+Compound Commands
+=================
+
+Cylc Provides compound commands which carry out more than one
+workflow action. For example Cylc provides a command to validate,
+install and play a workflow.
+
+Compound commands make common ways of working easier.
+
+.. note::
+
+   Use ``cylc command --help`` to get help for each compound command,
+   including a full list of available options.
+
+
+``cylc vip`` (Validate, Install and Play)
+-----------------------------------------
+
+Cylc VIP is the same as running:
```

Comment:
Could we give the path as explained in the example?

Reply:
like this?
```suggestion
``cylc vip /home/me/cylc-src/my-workflow`` is the same as running:
```